### PR TITLE
Rebased mongo 3

### PR DESCRIPTION
**Pull request by dlongley**
We should merge this to master instead of #73 -- but keep the branch for #73 around until a new release is in place and anyone depending on the branch has updated.

### CODE REVIEW

**Review by mattcollier — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # bedrock-ledger ChangeLog
 
+## 11.0.0 -
+
+### Changed
+- **BREAKING**: Use `bedrock-mongodb` 7.x-8.x.
+- Updated usage of MongoDB api to version 3.5.
+- Use `collection.{updateOne, updateMany}` over `collection.update`.
+- Pass `{projection}` to various methods as an option.
+
```
Comment (gannan08) [inline CHANGELOG.md:10]: We will need to add an update for the new consensus worker scheduler.

**Review by gannan08 — APPROVED** [review]
LGTM. We will need to ensure that the changelog captures the sum of all changes since the last release.

**Comment by aljones15** [comment]
thanks for doing this as rebasing https://github.com/digitalbazaar/bedrock-ledger-node/pull/73 with your work would have resulted with another project getting that code in production.

File: CHANGELOG.md (hunk 1 above)
Comment (dlongley) [reply CHANGELOG.md:13]: Added.

**Pull request merged**